.github/workflows/test.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       - name: Git checkout
         uses: actions/checkout@v3
       - name: install dependencies
-        run: yarn
+        run: yarn --immutable
```

Comment:
The snapshot was failing because here it was not just installing but updating dependencies, and a new version of MUI had a different ID for some component.